pipelines/tekton/azureml-container-pipeline/model-upload/local-model-pvc-template.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: template.openshift.io/v1
+kind: Template
```

Comment:
Given the rest of the repository uses Kustomize, is introduction of this OpenShift specific `Template` really needed?

Reply:
Yeah maybe using Kustomize is better, when assigned this it was always said to use `Template` but now that you say it I am not sure why

Reply:
If you have run-time variables / parameters in mind, then no: https://kubectl.docs.kubernetes.io/faq/kustomize/eschewedfeatures/#build-time-side-effects-from-cli-args-or-env-variables. The run should be reproducible with the content in the files, not something specified on a command line or in environment variables.

But you can of course override things with kustomize (including using patches or replacements), and that override of default can be as minimal as useful.

Reply:
I will set defaults in the Makefile for each variable similar to how I have done here. So now each option is optional when running the make target

https://github.com/opendatahub-io/ai-edge/pull/176/files#diff-b6d53e8f4a83a8b4900586623befbee132db0973ed649fb23ea48b0754f2528aR1-R5